src/Symfony/Component/Notifier/Bridge/RocketChat/README.md

```diff
@@ -11,8 +11,76 @@ ROCKETCHAT_DSN=rocketchat://ACCESS_TOKEN@default?channel=CHANNEL
 ```
 
 where:
- - `ACCESS_TOKEN` is your RocketChat access token
- - `CHANNEL` is your RocketChat channel
+ - `ACCESS_TOKEN` is your RocketChat Webhook URL
```

Comment:
Based on your example, this is not a URL in the DSN, but the access-token

Reply:
Rocket.Chat only provides a full URL in its integration panel, so I used this term to better align with this. But I agree this can be misleading. Maybe "webhook token" is clear enough.